### ISSUE TITLE
Warning, could not connect to Sentry native SDK

### ISSUE THREAD

**Issue by msqar** [getsentry/sentry-react-native#630]
OS:
- [ ]  Windows
- [X]  MacOS
- [ ]  Linux

_Platform:_
- [ ]  iOS
- [X]  Android

Output of `node -v && npm -v && npm ls --prod --depth=0`

```
v12.7.0
6.10.0
MyApp@0.0.1 /Users/marian-mac/Documents/dev/myapp
├── @babel/runtime@7.5.5
├── @react-native-community/async-storage@1.5.1
├── @react-native-community/geolocation@1.4.2
├── @react-native-community/slider@2.0.0
├── @sentry/react-native@1.0.0-beta.3
├── axios@0.19.0
├── buffer@5.2.1
├── lodash@4.17.15
├── native-base@2.13.0
├── prop-types@15.7.2
├── qs@6.7.0
├── react@16.8.6
├── react-native@0.60.4
├── react-native-actionsheet@2.4.2
├── react-native-animatable@1.3.2
├── react-native-gifted-chat@0.9.11
├── react-native-google-places-autocomplete@1.3.9
├── react-native-i18n@2.0.15
├── react-native-image-picker@1.0.1
├── react-native-keyboard-spacer@0.4.1
├── react-native-linear-gradient@2.5.5
├── react-native-loader@1.2.1
├── react-native-modal@11.1.0
├── react-native-modalbox@1.7.1
├── react-native-multistep-wizard@0.0.1
├── react-native-navigation@3.0.0-alpha.6
├── react-native-progress@3.6.0
├── react-native-snap-carousel@3.8.0
├── UNMET PEER DEPENDENCY react-native-svg@9.5.3
├── react-native-svg-uri-reborn@1.0.8
├── react-native-swipe-list-view@2.0.0
├── react-native-uuid@1.4.9
├── react-native-vector-icons@6.6.0
├── react-native-woodpicker@0.0.16
├── react-redux@7.1.0
├── redux@4.0.4
└── redux-thunk@2.3.0

npm ERR! peer dep missing: react-native-svg@^5.3.0, required by react-native-svg-uri-reborn@1.0.8
npm ERR! missing: native-base-shoutem-theme@github:GeekyAnts/theme#master, required by native-base@2.13.0
npm ERR! peer dep missing: eslint@>=3.14.1, required by eslint-config-prettier@6.0.0
```

Config:

```
import * as Sentry from '@sentry/react-native';

const store = configureStore();

Sentry.init({
    dsn: 'https://1302b466e1fa43e68c270faccabc1e93@sentry.io/1399438'
});
```

---
I have following issue:

On iOS there's no alert showing but when I open my app on Android, whenever the app starts, it throws the following message through Alert

"Warning, could not connect to Sentry native SDK. Did you forget to run react-native link @sentry/react-native?"

I already linked sentry, i even tried by running unlinking and then ` react-native link @sentry/react-native` again. It says `linked done` but error keeps showing.

Any idea on how to solve this?

**Comment by msqar** [comment]
I solved it by properly migrating to 0.60.4 and having autolinking working on both platforms. There was no other way. Thanks

**Comment by lucasluca** [comment]
having the same problem here
